activerecord/README.rdoc

```diff
@@ -195,6 +195,35 @@ Source code can be downloaded as part of the Rails project on GitHub:
 * https://github.com/rails/rails/tree/master/activerecord
 
 
+== Getting Started
+
+1.install sqlite to your PC
```

Comment:
Thank you for pointing out. 
I used double quote because the number list couldn't be interpreted correctly by rdoc.
I updated content.